solvers.tex

```diff
@@ -0,0 +1,127 @@
+%% Solvers in SymPy
+
+
+SymPy has a solvers module to solve equations symbolically. There are two
```

Comment:
_module of equation solvers for symbolic equations_ sounds better.
I would avoid repeating _solvers_, _solve_ two times.